docs/identity-platform/custom-extension-tokenissuancestart-configuration.md

```diff
@@ -390,6 +392,8 @@ Use the following steps to add Microsoft Entra as an identity provider to your A
 1. For the **Issuer URL**, enter the following URL `https://{domainName}.ciamlogin.com/{tenant_id}/v2.0`, where
     - `{domainName}` is the domain name of your external tenant, in the form `{domainName}.contoso.com`.
     - `{tenantId}` is the tenant ID of your external tenant.
+1. Under **Client application requirement** select **Allow requests from specific client applications** and enter `99045fe1-7639-4a75-9d4a-577b6ca3810f`.
```

Comment:
Also line L390 - Select **External configuration** as the tenant for your application

Reply:
fixed. also changed L391